stravalib/model.py

```diff
@@ -240,16 +355,49 @@ class LatLon(LatLng, BackwardCompatibilityMixin, DeprecatedSerializableMixin):
     """
 
     @root_validator
-    def check_valid_latlng(cls, values):
-        # Strava sometimes returns an empty list in case of activities without GPS
+    def check_valid_latlng(
```

Comment:
based on our conversation, i made this change to the types:

```
    def check_valid_latlng(
        cls, values: List[Optional[float]]
    ) -> Optional[List[float]]:
```
but am still getting this error

```
stravalib/model.py:380: error: Incompatible return value type (got "Optional[List[Optional[float]]]", expected "Optional[List[float]]")  [return-value]
```

so i guess there must be some way for n input value  of None here??